### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by zhongruige**
This issue closes the following by updating the README:

* https://github.com/woocommerce/woocommerce-e2e-boilerplate/issues/8
* https://github.com/woocommerce/woocommerce-e2e-boilerplate/issues/7
* https://github.com/woocommerce/woocommerce-e2e-boilerplate/issues/6

Changes including noting the required NPM version, notes on adding a `package.json` if one doesn't exist, and reorganizing the required packages.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,12 @@ This repo aims to provide a stripped down version of the default [WooCommerce E2
 
 ### Setup
 
-Note: the following assumes an existing WooCommerce extension with an existing `package.json`. 
+**Requirements**
+
+* Note that `npm@6` is required for setup
```
Comment (rrennick) [inline README.md]: We should include the command line

```
npm install -g npm@6
```

File: README.md [hunk 2]
```diff
@@ -30,7 +30,12 @@ This repo aims to provide a stripped down version of the default [WooCommerce E2
...
+* The following steps assumes there is an existing WooCommerce extension with an existing `package.json`. If your project does not contain a `package.json` file, create one using `npm init`. More information on this can be found [here](https://docs.npmjs.com/creating-a-package-json-file#creating-a-new-packagejson-file).
```
Comment (rrennick) [inline README.md]: For people who scan vs read documentation/instructions it's better to have steps they will take on their own lines

> If your project does not contain a `package.json` file, create one:

```
npm init
```

> More information on ...

File: README.md [hunk 3]
```diff
@@ -50,13 +55,13 @@ Note: the following assumes an existing WooCommerce extension with an existing `
 * Install required packages as development dependencies.
 
 ```bash
-npm install --save-dev @woocommerce/e2e-environment @woocommerce/e2e-utils jest
+npm i --save-dev @woocommerce/e2e-environment @woocommerce/e2e-utils @woocommerce/api @wordpress/e2e-test-utils jest
```
Comment (rrennick) [inline README.md]: `@wordpress/e2e-test-utils` no longer needs to be installed. All of it's functionality is exposed in `@woocommerce/e2e-utils`

**Review by rrennick — CHANGES_REQUESTED** [review]
@zhongruige The `api` is also listed as an optional package at https://github.com/woocommerce/woocommerce-e2e-boilerplate/tree/fix/readme-instruction-updates#optional-node-packages.

File: README.md (hunk 3 above)
Comment (zhongruige) [reply README.md]: Thanks! Given that, do you think it's still worth keeping this bullet point at the top of the document?

```
* [**WordPress E2E Utils**](https://www.npmjs.com/package/@wordpress/e2e-test-utils) - All of this package is exported from `@woocommerce/e2e-utils`. However, you can install it directly if you choose.
```

Or is it still useful for folks that may want to work with it directly? If so, I can add it to the optional install command below it, e.g.:

```bash
npm i --save-dev @woocommerce/e2e-core-tests @wordpress/e2e-test-utils
```

File: README.md (hunk 1 above)
Comment (zhongruige) [reply README.md]: Good call out! Done.

File: README.md (hunk 2 above)
Comment (zhongruige) [reply README.md]: Good suggestion, I'll break these out.

**Comment by zhongruige** [comment]
> @zhongruige The `api` is also listed as an optional package at https://github.com/woocommerce/woocommerce-e2e-boilerplate/tree/fix/readme-instruction-updates#optional-node-packages.

Thank you for the feedback @rrennick! I've re-worked this a bit. [I had one question](https://github.com/woocommerce/woocommerce-e2e-boilerplate/pull/13#discussion_r670521171) on if we should still include `@wordpress/e2e-test-utils` in the instructions (I left it in and added it to the optional command setup with my latest commit), so let me know your thoughts.

**Comment by rrennick** [comment]
@zhongruige It may be better to just remove the references to `@wordpress/e2e-test-utils`. If it's in both the project and our package then they need to keep the versions in sync to avoid having both versions installed.

**Comment by zhongruige** [comment]
> @zhongruige It may be better to just remove the references to `@wordpress/e2e-test-utils`. If it's in both the project and our package then they need to keep the versions in sync to avoid having both versions installed.

Good idea @rrennick, I've gone ahead and removed both references to `@wordpress/e2e-test-utils`.

**Comment by rrennick** [comment]
@zhongruige There are still some references left

```
$ grep -nR '@wordpress/e2e' ./
.//tests/e2e/config/jest.setup.example.js:7:} = require( "@wordpress/e2e-test-utils" );
.//README.md:98:* **Jest Setup Script** - `tests/e2e/config/jest.setup.js`. Can be used to define custom scripts for Jest. You can also make use of the example script **`tests/e2e/config/jest.setup.example.js`** which requires **`@wordpress/e2e-test-utils`** package to be installed to run the defined functions. Please note that once the default template is enabled, all posts and products will be trashed, and local storage will be cleared between tests.
.//README.md:106:* **WordPress E2E Test Utils** - https://www.npmjs.com/package/@wordpress/e2e-test-utils
```

In the jest setup script it can be changed to `@woocommerce/e2e-utils`.

**Comment by zhongruige** [comment]
> `@woocommerce/e2e-utils`

@rrennick good catch, removed those as well.

**Comment by rrennick** [comment]
> @rrennick good catch, removed those as well.

@zhongruige Did you push those? The readme is the only changed file on the PR.

**Comment by zhongruige** [comment]
> > @rrennick good catch, removed those as well.
> 
> @zhongruige Did you push those? The readme is the only changed file on the PR.

Thanks @rrennick, had it ready to go but did miss pushing it. It's updated now 👍

**Review by rrennick — APPROVED** [review]
@zhongruige Nice work.

**Pull request merged**